### DISCUSSION TITLE
SSG - 0 Html files pre rendered.

### DISCUSSION THREAD

**Discussion by techniath** [vikejs/vike#586] [Discussion]
Hey 👋, I am a newbie with UI development. I was reading about ssr and came across ssg with vite. So, I followed pre rendering article (I don't have dynamic routes yet..) so just added the ssr pre render true in vite.config and tried running vite build. 
I am getting 0 html documents pre-rendered. 
Note: I am using vue and under pages I have directories
Pages
-index.vue
-about
--index.vue
Etc ..

Edit:
1) I updated the index.vue page to index.page.vue and did vite build. I am now getting an error.
![image](https://user-images.githubusercontent.com/74625768/210291999-4c95fc88-5b3e-47ed-ba4f-800ff32284df.png)

Thank you for the help.

**Comment by brillout** [comment]
Do you use `import.meta.glob`? Feel free to publish a reproduction and I'll have a look at it.

**Reply by techniath** [reply]
Yes, I use import.meta.glob

**Reply by techniath** [reply]
I will upload the repo. Also, I tried to clone and run the [https://github.com/brillout/vite-plugin-ssr/tree/dev/examples/vue-full](https://github.com/brillout/vite-plugin-ssr/tree/dev/examples/vue-full) but I am facing this error.
![image](https://user-images.githubusercontent.com/74625768/210661741-6141416f-2abe-45af-990f-149b163e1968.png)
Is it related to [invalid-esm](https://vite-plugin-ssr.com/invalid-esm#solution)

**Reply by brillout** [reply]
Reproduction welcome as well for that last error.
